src/components/plots/linePlot.ts

```diff
@@ -23,7 +23,7 @@ export module Plots {
       this._defaultStrokeColor = new Scales.Color().range()[0];
     }
 
-    protected _rejectNullsAndNaNs(d: any, i: number, dataset: Dataset, plotMetadata: any, accessor: _Accessor) {
+    protected _rejectNullsAndNaNs(d: any, i: number, dataset: Dataset, plotMetadata: any, accessor: Accessor<X> | Accessor<number>) {
```

Comment:
This function has a goofy signature. Is it possible to just inline it where it's used?